lib/sedna/backend/tensorflow/__init__.py

```diff
@@ -30,6 +31,19 @@
     Session = tf.Session
 
 
+_url = os.getenv("S3_ENDPOINT_URL", "http://s3.amazonaws.com")
```

Comment:
1. please put these s3 codes into a function instead of top level
2. why only does tensorflow have this config?

Reply:
1. fix
2. follow https://docs.w3cub.com/tensorflow~guide/deploy/s3